README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portafolio
 
-> I create my first Portfolio the set up and mobile first
+> I Create firts Portafolio Templates Mobiles
```

Comment:
- [OPTIONAL] Please add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. As not requested by the previous reviewer, I consider this optional but highly recommended.

Reply:
ok i edith it